app/assess/routes.py

```diff
@@ -72,3 +72,30 @@ def application(fund_id: str, round_id: str, application_id: str):
         round=fund_round,
         application=application,
     )
+
+
+@assess_bp.route(
+    "/view_application/<fund_id>/<round_id>/<application_id>", methods=["GET"]
+)
+def view_application(application_id, fund_id, round_id):
+
+    fund_data = get_fund(fund_id)
+    if not fund_data:
+        abort(404)
+
+    round_data = get_round(fund_id=fund_id, round_id=round_id)
+    if not round_data:
+        abort(404)
+
+    application_data = get_application(
+        fund_id=fund_id, identifier=application_id
+    )
+    if not application_data:
+        abort(404)
+
+    return render_template(
+        "assesser_project_view.html",
```

Comment:
Could this template nam be simplified to match the route name a bit more closely? So it's easy to identify which templates go with which routes when we have lots of them in future? - appreciate there's a conflict with the legacy view of last sprint, but could something like "project_summary" work?